CHANGELOG.md

```diff
@@ -5,6 +5,24 @@
 * [#128](https://github.com/elastic/helm-charts/pull/128) - @Crazybus - Add ci jobs for metricbeat
 * [#127](https://github.com/elastic/helm-charts/pull/127) - @Crazybus - WIP add metricbeat chart
 
+---
+## 7.1.1 - 2019/06/07
+
+* 7.1.1 as the default stack version
+* Helm 2.14.0 as the tested version. Helm 2.14.0 has some extra validation built in which caused an issue with an [invalid field in the filebeat chart](https://github.com/elastic/helm-charts/issues/136).
+
+### Elasticsearch
+
+* [#146](https://github.com/elastic/helm-charts/pull/146) - @Crazybus - Add instructions for how to enable snapshots
+
+### Kibana
+
+* [#151](https://github.com/elastic/helm-charts/pull/151) - @natebwangsut - Added an option to add annotations(s) to service resource
```

Comment:
It would be a good idea to use the same tenses in all the comments: `Added` -> `Add`, like this is done above and below but this is just a general comment and not a showstopper

Reply:
Yeah I normally do, this one was directly copy pasted from the pull request itself. I normally only rewrite them if the message isn't clear enough. But small changes like this should be fine!